service-templates/apigw-lambda-svc/dev-resources/proton.auto.tfvars.json

```diff
@@ -1,48 +0,0 @@
-{
```

Comment:
the files in `dev-resources/` have been replaced with a combination of the `varaibles.tf` file and `terraform.tfvars`. `terraform.tfvars` is used somewhat like `.env` files in that its not checked in and is gitignored, but a reference file has been provided to copy+paste. Any variables provided in `variables.tf` can be set in the `terraform.tfvars` and will automatically be picked up for use when running `terraform plan` or `terraform apply`